src/siotls/examples/minih1s.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+"""
+Mini HTTP/1.1 server over TLS with openssl, used to test the siotls
+client against another TLS implementation.
+"""
+
+__file_path__ = __import__('pathlib').Path(__file__)  # eventually standard??
+
+import logging
+import socket
+import ssl
+import textwrap
+import sslkeylog
+
+HOST = '127.0.0.1'
+PORT = 8446
+
+logging.basicConfig(level='INFO')
+logger = logging.getLogger(__file_path__.stem)
+
+sslkeylog.set_keylog('/home/julien/.tlskeylogfile')
+context = ssl.SSLContext(ssl.PROTOCOL_TLS_SERVER)
+context.set_alpn_protocols(['http/1.1'])
+context.load_cert_chain(
+    __file_path__.joinpath('../../data/self-signed-cert.pem').resolve(),
+    __file_path__.joinpath('../../data/self-signed-key.pem').resolve(),
+)
+
+with socket.socket(socket.AF_INET, socket.SOCK_STREAM, 0) as sock:
+    sock.bind((HOST, PORT))
+    sock.listen(5)
+    logger.info("listening on %s port %s", HOST, PORT)
+    with context.wrap_socket(sock, server_side=True) as server:
+        try:
+            while True:
+                client = None
+                try:
+                    client, client_info = server.accept()
+                except Exception:
+                    logger.exception("while accepting incomming connection")
+                    continue
+                logger.info("new connection from %s", client_info[1])
+                try:
+                    client.recv(16384)
+                    client.send(textwrap.dedent("""\
+                        HTTP/1.1 204 No Content
+                        Connection: close
```

Comment:
missing mandatory Date header https://httpwg.org/specs/rfc9110.html#rfc.section.6.6.1

Reply:
also Server should be present
suggestion:

    minih1s OpenSSL/{ssl.OPENSSL_VERSION_INFO} Python/{sys.version_info}